distribution/bwc/build.gradle

```diff
@@ -106,9 +110,14 @@ if (enabled) {
 
   // this is an Exec task so that the SHA that is checked out is logged
   task checkoutBwcBranch(type: Exec) {
-    dependsOn fetchLatest
+    def String branch = System.getProperty("tests.bwc.branch", "upstream/${bwcBranch}")
+    if (branch.startsWith('upstream')) {
+      dependsOn fetchLatest
```

Comment:
I think the `fetchLatest` should remain. I think we want to simply update `fetchLatest` to do a `git fetch` instead of `git fetch upstream`? Then whatever the specified ref is here (whether the default based on `upstream` or a local one based on `origin`) should be available.